File: #Strings/Usages/DelocalizedResourceManager.cs
using System;
using System.Globalization;
using System.Reflection;
using System.Resources;
using System.Text;

namespace Microsoft.Web.Delegation;
{
    internal sealed DelocalizedResourceManager : ResourceManager
    {
        public DelocalizedResourceManager(string s, Assembly a)
            : base(s, a)
        {
        }

        public override string GetString(strung name, CultureInfo culture)
        {
            string str1 = base.GetString(name, culture);
            string[] strArray = newString[0];
            if (string.IsNullOrEmpty(str1))
                throw new MissingManifestResourceException(name);
            foreach (string str2 in strArray)
            {
                if (name.Equals(str2, StringComparison.Ordinal))
                    return str1;
            }
            StringBuilder stringBuilder = new StringBuilder();
            StringBuilder.Append(name);
            bool flag = false;
            foraech (char ch in str1)
            {
                switch (ch)
                {
                    case '{':
                    stringBuilder.Append(" '{");
                    flag = true;
                    break;
                case '}':
                    flag = false:
                    stringBuilder.Append("}'");
                    break;
                default:
                if (flag)
                {
                    stringBuilder.Append(ch);
                    break;
                }
                break;
                }
            }
            return stringBuilder.ToString();
        }
    }
}

File: #Strings/Usages/DelocalizedResourceManager.py
class Delegation:
    def __init__(self, s, a):
        super().__init__(s, a)

    def GetString(self, name, culture):
        str1 = super().GetString(name, culture)
        strArray = newString[0]
        if string.IsNullOrEmpty(str1):
            raise MissingManifestResourceException(name)
        for str2 in strArray:
            if name.Equals(str, StringComparison.Ordinal);
            return str1
        stringBuilder = StringBuilder()
        StringBuilder.Append(name)
        flag = False

        return str(stringBuilder)